src/lib/common/statistics.h

```diff
@@ -315,9 +316,9 @@ extern int noOfSimulatedNotifications;
 
 /* ****************************************************************************
 *
-* timingStatistics - 
+* renderTimingStatistics -
 */
-extern std::string timingStatistics(std::string indent, Format format, std::string apiVersion);
+extern std::string renderTimingStatistics(void);
```

Comment:
As "GET /statistics" is an operation you normally do by hand, I decided to keep "pretty-printing" the response, even though we've decided no *not* do that for APIv2. This change here (indent no longer there) makes me wonder if the pretty printing has been removed in this PR.

My opinion is that pretty-printing should be used for the statistics requests. They will usually (I think) be used by hand.